docs/collections/documents/tutorials/bulk-update-restql.md

```diff
@@ -399,30 +485,65 @@ async function restqldemo() {
     /* Execute RestQl Queries */
     console.log("------- EXECUTING THE QUERIES  ------");
 
+    await sleep(2000);
     await client.executeRestql("insertData", {
-        InputDocs: inputDocs
+      InputDocs: inputDocs
     });
 
     console.log("Data Inserted \n");
 
-    const res = await client.executeRestql("getData");
+    let res = await client.executeRestql("getData");
 
     console.log("Output of get data query:");
     console.log(res.result);
     console.log("\n");
 
     await client.executeRestql("updateData", {
-        updateKeys,
-        updateKeyValue
+      updateKeys,
+      updateKeyValue
     });
 
     console.log("Data updated \n");
 
-    const data = await client.executeRestql("getData");
+    res = await client.executeRestql("getData");
 
     console.log("Output of get data query after update:");
-    console.log(data.result);
+    console.log(res.result);
+    console.log("\n");
+
+    await client.updateRestql("insertData", updatedInsertData, {});
+    await sleep(2000);
+
+    await client.executeRestql("insertData", {
+      barryKey
+    });
+
+    console.log("New Data Inserted \n");
+
+    res = await client.executeRestql("getData");
+
+    console.log("Output of get data query after insertion of new data:");
+    console.log(res.result);
     console.log("\n");
+
+    res = await client.getRestqls();
+
+    console.log("List of all available Saved Queries: ");
```

Comment:
```suggestion
    console.log("List of all available saved queries: ");
```

Reply:
Do you mean query workers?

Reply:
yes basically Saved queries is same as query workers which is same as RestQl .. you can let me know which term would you like us to use here